spec/jobs/algolia/remove_from_index_job_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "rails_helper"
+require "jobs/shared_examples/enqueues_job"
+
```

Comment:
We're not testing what the job does, just its interface. Would you mind adding some testing of the inside? Have a look at `spec/jobs/articles/update_analytics_job_spec.rb` for inspiration. You can mock the algolia object and verify it has been called

Reply:
More tests won't harm, but in this case the test will probably look like stub everything and test (almost) nothing.

Reply:
You have a point, isn't there a testing mode for the algolia object?

Reply:
I suppose there is a way to test this.